### PR TITLE
[TC-617] remove division table

### PR DESCRIPTION
**Pull request by chelsea-EYDS**
[TC-0617](https://bcdevex.atlassian.net/browse/TC-0617)

Objective:

- remove division table
- set division as optional string on personnel table
- move ministry back to personnel table

### CODE REVIEW

File: backend/src/common/bcws-seed.ts [hunk 1]
```diff
@@ -97,6 +94,8 @@ export const handler = (
     homeLocation: faker.helpers.arrayElement(locations),
     workLocation: faker.helpers.arrayElement(locations),
     firstName: faker.person.firstName(),
+    division: faker.company.name(),
```
Comment (fw-noel) [inline backend/src/common/bcws-seed.ts:97]: I might still suggest keeping this from an array of enums, just to keep it realistic? 

If it's too much of a pain, all good

File: backend/src/database/migrations/1717446743416-supervisor-migration.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class SupervisorPhoneMigration1717446743416 implements MigrationInterface {
+    name = 'SupervisorPhoneMigration1717446743416'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "personnel" ADD "supervisor_phone" character varying(10)`);
+        await queryRunner.query(`ALTER TABLE "personnel" ALTER COLUMN "ministry" SET NOT NULL`);
```
Comment (fw-noel) [inline backend/src/database/migrations/1717446743416-supervisor-migration.ts:8]: Can we move this to the bottom of the migration above?

File: frontend/src/pages/profile/ProfileEditForm/constants.ts [hunk 3]
```diff
@@ -60,9 +60,17 @@ export const editProfileValidationSchema = Yup.object().shape({
     .max(50, 'Max length 50 characters.')
     .required('This field is required.'),
   supervisorEmail: Yup.string().optional().email('Invalid email format.'),
-
+  supervisorPhone: Yup.string()
+    .optional()
+    .test(
+      'phone number',
+      'Invalid phone number format. Please enter ten digits.',
```
Comment (fw-noel) [inline frontend/src/pages/profile/ProfileEditForm/constants.ts:67]: `enter ten digits without formatting`? Or maybe an example?

**Review by fw-noel — DISMISSED** [review]
I don't believe this is yet dropping the divisions table, is it?

**Comment by chelsea-EYDS** [comment]
> I don't believe this is yet dropping the divisions table, is it?

lol, nope!

**Review by fw-noel — APPROVED** [review]

**Pull request merged**